new-ops/TEAL/acct_params_get_demo.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Acct Params Get Demo",
+    "description": "A demo application to show off some of the new AVM 1.1 opcodes. These include: `acct_params_get`, with the fields `AcctBalance`, `AcctMinBalance`, and `AcctAuthAddr`.",
+    "networks": {},
+    "methods": []
```

Comment:
no methods?

Reply:
I'm using bare application calls here, I had a look in the spec but didn't quite see how they should be referenced. This script doesn't do anything really, but the demo is within the optin and noop bare application calls.